draft-ietf-quic-transport.md

```diff
@@ -3271,15 +3257,35 @@ Header Form:
 : The most significant bit (0x80) of byte 0 (the first byte) is set to 1 for
   long headers.
 
-Long Packet Type:
+Fixed Bit:
+
+: The next bit (0x40) of byte 0 is set to 1.  Packets containing a zero value
+  for this bit are not valid packets in this version.
```

Comment:
Should we say that those packets are supposed to be dropped?

Reply:
@marten-seemann I believe there a general section on dropping invalid handshake packets. But the following paragraphs says to raise a protocol violation - which only makes sense if the packet is authenticated. Either way, I think this paragraph needs to be consistent with the other fields.

Reply:
For consistency, I would say that receipt of zero here should be treated  as a connection error of type PROTOCOL_VIOLATION.

Reply:
@janaiyengar: I was thinking of a fast-path processing here: I can check this bit without decrypting the packet, so if I receive a packet with 0x40 == 0, I know it's some UDP garbage, and I can drop it on the floor immediately.

Reply:
Yeah we have two choices:
* drop the apparently broken packet without even decrypting it
* decrypt and verify the AEAD tag of the apparently broken packet, then close the connection with PROTOCOL_VIOLATION

I tend to agree what @marten-seemann that we do not need to decrypt in this case. Let's handle it the same way as handling a packet with a broken AEAD tag.

Reply:
Not quite the same way.  This can be detected using public information, so you don't need to run a constant-time decryption on that packet.